tests/bud/dockerignore3/.dockerignore

```diff
@@ -0,0 +1,9 @@
+# comment
+*
+!test*
+!src
+**/*.in
+*.md
+!README*.md
+README-secret.md
+test1.txt
```

Comment:
I'm not sure this is covered elsewhere, but could you create a `src/etc` directory with a couple of files in it and add 'src/etc to the end here and make sure those files don't get in?  Or do you think you've that scenario covered and I'm missing it?

Reply:
It certainly can't hurt.  Added.